iothub/device/src/IotHubModuleClient.cs

```diff
@@ -87,14 +87,20 @@ internal IotHubModuleClient(IotHubConnectionCredentials iotHubConnectionCredenti
         /// <returns>A disposable client instance.</returns>
         public static async Task<IotHubModuleClient> CreateFromEnvironmentAsync(IotHubClientOptions options = default)
         {
-            // Make sure client options is initialized.
-            if (options == default)
-            {
-                options = new();
-            }
+            IotHubClientOptions clientOptions = options != null
+                ? new ()
+                {
+                    FileUploadTransportSettings = options.FileUploadTransportSettings,
```

Comment:
Is this used anywhere? Modules can't upload files, but they can make http requests to invoke methods